web/src/features/fireBehaviourCalculator/components/FBCResultTable.tsx

```diff
@@ -103,6 +106,20 @@ const FBCResultTable = (props: Props) => {
                 <TableCell>BUI</TableCell>
                 <TableCell>FWI</TableCell>
                 <TableCell>HFI</TableCell>
+                {/* <TableCell>
+                  Critical
+                  <br />
+                  Hours
+                  <br />
+                  (4000 HFI)
```

Comment:
Can we delete this or is it temporarily commented out?

Reply:
just temporary - will be added back in soon